src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
As I mentioned before, we can just ask for sepecifc things from lib and events. For example:
```js
import { createEl, bind } from './lib.js';
import { on, off } from './events.js';
```